adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VCFInFormatter.scala

```diff
@@ -56,7 +59,8 @@ private[variant] case class VCFInFormatter private (
   protected val companion = VCFInFormatter
 
   // make a converter
-  val converter = new VariantContextConverter(Some(sequences))
+  val converter = new VariantContextConverter(headerLines,
+    ValidationStringency.LENIENT)
```

Comment:
the methods seem to default to strict, and the args default to strict, so I'm confused about the usage here.